coremltools/test/neural_network/test_compiled_model.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2023, Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can be
+# found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+from shutil import copytree, rmtree
+from tempfile import TemporaryDirectory
+
+from unittest import TestCase
```

Comment:
We have been slowly trying to get rid of unittest and just use pytest. Would be good to use pytest instead

Reply:
Oh, I wasn't aware of that. I'll switch this to pytest.

What's the reason for preferring pytest over unitest?